File: 1.Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hmap = {}
        
        for i,v in enumerate(nums):
            diff = target - v
            if diff in hmap:
                return [i, hmap[diff]]
            else:
                hmap[v]=i

File: 1-two-sum/1-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        Dictionary < int, int > map = new Dictionary < int, int > ();
        
        var res = new int[2];
        for (int i=0; i<nums.Length; i++ ){
            var diff = target - nums[i];
            
            if (map.ContainsKey(diff)){
                int[] res1 = {map[diff],i};
                return res1;
            }
            else{
                map[nums[i]]=i;
            }
        }
        return res;
    }
}